dir.props

```diff
@@ -62,10 +62,13 @@
   <!-- Import Build tools common props file where repo-independent properties are found -->
   <Import Project="$(ToolsDir)Build.Common.props" Condition="Exists('$(ToolsDir)Build.Common.props')" />
 
-  <!-- Enable the analyzers for this repo -->
+  <!-- Enable the analyzers for this repo, except the PinvokeAnalyzer -->
   <PropertyGroup>
     <EnableDotnetAnalyzers Condition="'$(EnableDotnetAnalyzers)'==''">true</EnableDotnetAnalyzers>
   </PropertyGroup>
+  <ItemGroup Condition="'$(EnableDotnetAnalyzers)' == 'true'">
```

Comment:
Can you please file an issue to re-enable this and make it run against CoreFX and CoreCLR repos?